extensions/elytron-security-common/deployment/src/main/java/io/quarkus/elytron/security/common/deployment/QuarkusSecurityCommonProcessor.java

```diff
@@ -22,6 +22,17 @@ public void registerPasswordProvider(ElytronCommonRecorder recorder) {
         recorder.registerPasswordProvider();
     }
 
+    /**
+     * Graal VM now seems to loose providers registered at static init
```

Comment:
Linking https://github.com/quarkusio/quarkus/issues/17046#issuecomment-879533170 for future reference